src/main/kotlin/org/rust/lang/utils/RustEscapesUtils.kt

```diff
@@ -32,7 +32,7 @@ fun parseRustStringCharacters(chars: String, outChars: StringBuilder, sourceOffs
 
     val outOffset = outChars.length
     var index = 0
-    chars.tokenize(RustEscapesLexer(STRING_LITERAL, true, true)).forEach {
+    chars.tokenize(RustEscapesLexer(STRING_LITERAL, true, true, true)).forEach {
```

Comment:
Linking to the diff was probably a bad idea. I am talking about this two lines

https://github.com/jajakobyly/intellij-rust/blob/lex-fix/src/main/kotlin/org/rust/lang/core/lexer/RustEscapesLexer.kt#L125
https://github.com/jajakobyly/intellij-rust/blob/lex-fix/src/main/kotlin/org/rust/lang/utils/RustEscapesUtils.kt#L35